environment.yml

```diff
@@ -15,16 +16,30 @@ dependencies:
   - numpy
   - pandas
   - pip
-  - pyerfa
+  - pyerfa=1.7.3
   - pytables
   - pytest
   - pytest-cov
   - python=3.9
   - pyyaml
+  - scikit-hep>3.1,<3.3
   - scikit-learn
   - scipy
   - tqdm
   - twine
   - wheel
+  # for zfit
+  - iminuit
+  - nlopt
+  - numdifftools
+  - tensorflow
+  - tensorflow-probability
```

Comment:
```suggestion
```
remove as they are not up to date

Reply:
This installs 2.4.2, do you really need the very latest 2.5 from pypi?

Reply:
That will only be true for the upcoming zfit release with Python 3.9 support, if I'm not mistaken. But @mayou36 can confirm.

Reply:
If it helps we can go to 3.8 for the moment.

Reply:
Yes, this will be needed. Unfortunately (or also fortunately), tensorflow still has quite a lot of breaking changes in their API and therefore 2.5 is needed.

For the release, I think in 30 mins both zfit (already out) and hepstats should work with 3.9

Reply:
I just got the report from an apple user, that 3.9 is required to run on M1. So we might have a problem here.

Do we really need zfit / phasespace in the environment? Or would a demo be enough for the school?

Reply:
I would say yes, but it should not be a problem, they all run in 3.9 (the CIs just passed). We simply need the hepstats pip release (~15 mins and we have it) and it should work